taskomatic/jobs/cacheman.py

```diff
@@ -42,8 +45,7 @@ def _materialize_cve_cache(cur, account_id, account_name, current_cache):
                                         sp.rh_account_id = %s AND
                                         sp.opt_out = false AND
                                         sp.stale = false AND
-                                        sp.when_deleted IS NULL AND
-                                        coalesce(sp.host_type, 'null') != 'edge') INNER JOIN
+                                        sp.when_deleted IS NULL{host_type_cond}) INNER JOIN
```

Comment:
https://www.psycopg.org/docs/sql.html#module-psycopg2.sql , you could use this, instead of F-string

Reply:
changed, thanks!